### DISCUSSION TITLE
[Bug]: Feather icons 1.2.0 viewBox production / author field

### DISCUSSION THREAD

**Discussion by HowToMeetLadies** [GetPublii/Publii#1374] [Bug Report]
### Operating system

macOS

### Publii version

0.44.4

### Post editor

None

### Bug description

The plugin doesn't output a ViewBox for me. Without this information, the symbols will not be rescaled in production (Safari). Why exactly is this property only added in preview mode? Or what am I doing wrong?

I patched the plugin to output viewBox and the icons were scaled correctly.

---
The “Plugin Author” field is still “Your Name”.

### Steps to reproduce

Version 0.44.4
Feather icons 1.2.0, defaults/internal
Terminal theme
[fi=award size=256px]

**Comment by HowToMeetLadies** [comment]
I removed the plugin and reinstalled it (from the same ZIP file) and it still doesn't output a viewBox. But Safari now displays the icon correctly. nm

Safari criticizes the use of "rem":
<img width="324" alt="rembrandt" src="https://github.com/GetPublii/Publii/assets/755996/c4fc9e61-1ccb-43b4-b272-945fd27b2e93">

---

Many thanks to the team at Publii - Feather is such a great collection.

**Reply by bobmitro** [reply]
The lack of the viewBox attribute means the SVG will not adapt flexibly to the container's size. However, it is inconsequential because the width and height attributes define the icon's size. I will include it in the plugin's next release to ensure consistency. 
The viewBox size is specified in pixels, but other units, including rem, can also be utilized. 

When specifying dimensions for an SVG in HTML or CSS using rem units, most browsers convert these values to pixels based on the root element's font size. However, Safari may not recognize rem units for SVG dimensions specified directly in the SVG's width and height attributes, as it adheres strictly to the SVG specification. Despite this, the icon size is still displayed correctly; the only issue is an error message shown in the console.
